modules/core/Sampler.cpp

```diff
@@ -107,6 +107,16 @@ namespace rt {
 		return Vec3f(s[0], s[1], z);
 	}
 
+	Vec2f CSampler::uniformSampleRegularNgon(const Vec2f& sample, int n, int m) {
```

Comment:
Maybe it makes sense to check whether n is equal to 0., _e.g._
`If (n == 0) return uniformSampleDisk(...);` as the first string

Reply:
yes doing this check in uniformSampleRegularNgon() would be better instead of putting it in CCameraPerspective (to keep the code cleaner)